modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json

```diff
@@ -501,7 +501,7 @@
     },
     {
       "id": "dog_EVA_PRJEB24066",
-      "species": "canis_familiaris",
+      "species": "canis_lupus_familiaris",
```

Comment:
Do these species in the vcf_config.json change also need to be made on branch 100?

Reply:
Yes, it should have been. Do we want to do it retroactively?